index.js

```diff
@@ -0,0 +1,3 @@
+'use strict';
+
+module.exports = exports = require('./dist/jmap-client.min.js');
```

Comment:
`exports` starts as a reference to `module.exports`, but if you remap `module.exports` then `exports` is outdated. Thus the convention is to remap both as I did...